templates/post-multiple-authors.adoc

```diff
@@ -22,6 +23,9 @@ AUTHOR_NAME <https://github.com/GITHUB_USERNAME>
 // Above:
 // Do not insert any blank lines between any of the lines above.
 //
+// "open-graph-image" is set to OL logo. Whenever possible update this to a more appriopriate/specific image (For example if present a image that is being used in the post). However, it
```

Comment:
"an image" etc as for the beta template.

Reply:
Done